uvm/impl/com/untangle/uvm/EventManagerImpl.java

```diff
@@ -60,7 +60,7 @@
  */
 public class EventManagerImpl implements EventManager
 {
-    private static final Integer SETTINGS_CURRENT_VERSION = 3;
+    private static final Integer SETTINGS_CURRENT_VERSION = 4;
```

Comment:
EventSettings.java has a default version as well.

Reply:
Updated the version in EventSettings